src/Simplex/Messaging/Agent.hs

```diff
@@ -163,6 +165,27 @@ processCommand c@AgentClient {sndQ} (corrId, connAlias, cmd) =
       where
         ackMsg rq = sendAck c rq >> respond OK
 
+    suspendConnection :: m ()
+    suspendConnection =
+      withStore (`getConn` connAlias) >>= \case
+        SomeConn _ (DuplexConnection _ rq _) -> suspend rq
+        SomeConn _ (ReceiveConnection _ rq) -> suspend rq
```

Comment:
`suspend rq` is so small I'm not sure it's worth having it as a separate function, might be more readable just duplicating a couple of calls

Reply:
small functions is good - easier to add things in the future. I am starting to create them easier - it's easier than refactor large functions later